examples/terraform/nutanix/variables.tf

```diff
@@ -148,9 +158,32 @@ variable "bastion_disk_size" {
   type        = number
 }
 
+variable "worker_vcpus" {
+  default     = 2
+  description = "Number of vCPUs per socket for worker nodes"
+  type        = number
+}
+
+variable "worker_sockets" {
+  default     = 1
+  description = "Number of sockets for worker nodes"
+  type        = number
+}
+
+variable "worker_memory_size" {
+  default     = 4096
+  description = "Memory size, in Mib, for worker nodes"
+  type        = number
+}
+
+variable "worker_disk_size" {
+  default     = 50
+  description = "Disk size size, in Gb, for worker nodes"
+  type        = number
+}
+
 variable "initial_machinedeployment_replicas" {
   default     = 1
   description = "number of replicas per MachineDeployment"
   type        = number
-
 }
```

Comment:
Would be good to regenerate terraform-docs section of the README

Reply:
@kron4eg Done, PTAL.